jobs/opensearch/templates/config/opensearch-security/roles_mapping.yml.erb

```diff
@@ -69,6 +59,12 @@ readall:
   - "smoketest.opensearch.internal"
   backend_roles:
   - "readall"
+  - "auditor"
+
+auditor_access:
```

Comment:
What does auditor_access role provide that `readall` doesn't already provide?

Reply:
makes a tenant for them to use